frontend/src/metabase/routes.jsx

```diff
@@ -139,7 +139,7 @@ export const getRoutes = store => {
 
           <Route path="search" title={t`Search`} component={SearchApp} />
           {/* Send historical /archive route to trash - can remove in v52 */}
-          <Redirect path="archive" to="trash" replace />
+          <Redirect from="archive" to="trash" replace />
```

Comment:
This isn't related to the current PR, but I noticed that this prop is using the wrong name, so I've fixed it on the fly here.